src/CommentSection.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import { findDOMNode } from 'react-dom';
+
+class CommentSection extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            comments: []
+        };
+    }
+
+    componentDidMount() {
+        this.setState({
+            comments: this.props.comments
+        });
+    }
+
+    handleSubmit = (e) => {
+        e.preventDefault();
+        const text = findDOMNode(this.refs.commentInput).value;
+        const newComment = {
+            username: 'Sean',
+            text
+        };
+        this.setState({
+            comments: [...this.state.comments, newComment]
+        });
+        findDOMNode(this.refs.commentInput).value = '';
```

Comment:
There are better ways to handle selecting an element from within a react component. By better I just mean more commonly seen today as best practices.